Week 4/Generate Parenthesis/Solution.java

```diff
@@ -0,0 +1,40 @@
+import java.io.*;
```

Comment:
Wow! You went above and beyond this week by tackling the bonus problems in addition to your homework! Awesome job :)